webpack.config.js

```diff
@@ -39,21 +39,28 @@ function convertEmojiToolkitCodePointToChar(codePoint) {
 
 function jsonTransform(emojis) {
   const result = {};
+
   for (const emojiData of Object.values(emojis)) {
     const char = convertEmojiToolkitCodePointToChar(emojiData.code_points.fully_qualified);
     const data = {
       char,
       slug: emojiData.shortname.replace(/:/g, ''),
+      category: emojiData.category,
     };
+
     result[data.slug] = data;
+
     for (const alternativeShortName of emojiData.shortname_alternates) {
       // :tf: is a legacy betterttv global emote
       if (alternativeShortName === ':tf:') {
         continue;
       }
+
+      data.isAlternative = true;
       result[alternativeShortName.replace(/:/g, '')] = data;
```

Comment:
im not sure this works as you intend, and potentially only emotes without alternatives are rendering.

data on line 60 is equal to data on line 51, which means data is also equal in the resulting `result`